windows/src/engine/keyman32/kmhook_getmessage.cpp

```diff
@@ -233,6 +234,13 @@ LRESULT _kmnGetMessageProc(int nCode, WPARAM wParam, LPARAM lParam)
 		return CallNextHookEx(Globals::get_hhookGetMessage(), nCode, wParam, lParam);
 	}
 
+  if (mp->message == wm_keyman_control)
+  {
+      SendDebugMessageFormat(0, sdmInternat, 0, "GetMessage: wm_keyman_control hwnd=%x %x %x", mp->hwnd, mp->wParam, mp->lParam);
+      ProcessWMKeymanControl(mp->wParam, mp->lParam);
+      return CallNextHookEx(Globals::get_hhookGetMessage(), nCode, wParam, lParam);
+  }
+
```

Comment:
New code should probably match the new coding style ... although I am not 100% sold on mixing brace style. Certainly indents should be 2 chars though!

```suggestion
  if (mp->message == wm_keyman_control) {
    SendDebugMessageFormat(0, sdmInternat, 0, "GetMessage: wm_keyman_control hwnd=%x %x %x", mp->hwnd, mp->wParam, mp->lParam);
    ProcessWMKeymanControl(mp->wParam, mp->lParam);
    return CallNextHookEx(Globals::get_hhookGetMessage(), nCode, wParam, lParam);
  }

```